tests/test_fsdp_examples.py

```diff
@@ -126,6 +127,7 @@ def _test_fsdp(
             "--use_flash_attention True",
             "--flash_attention_causal_mask True",
             f"--token {token.value}",
+            "--sdp_on_bf16",
```

Comment:
I see that this is common for both if/else case. Can we move it to the common command after line 83?